### PR TITLE
Referral report

### PR DESCRIPTION
**Pull request by kkrampa**
http://manage.dimagi.com/default.asp?262732#1398937
@calellowitz

### CODE REVIEW

**Comment by kkrampa** [comment]
@calellowitz Could you merge this PR?

File: custom/enikshay/ucr/data_sources/person_for_referral_report_v2.json [hunk 1]
```diff
@@ -1323,6 +1322,254 @@
...
+            },
+            {
+                "display_name": "episode_type",
+                "transform": {},
+                "datatype": "string",
+                "type": "expression",
+                "is_primary_key": false,
+                "is_nullable": true,
+                "column_id": "episode_type",
+                "expression": {
```
Comment (calellowitz) [inline custom/enikshay/ucr/data_sources/person_for_referral_report_v2.json]: i dont think this meets the spec. it looks like you should display `Confirmed DSTB (Patient) ` for `confirmed_tb` and `Confirmed DRTB (Patient)` for `confirmed_drtb`. additionally, i spoke with sarvesh and it seems there should be a third option `Presumptive TB` for `presumptive_tb`

File: custom/enikshay/ucr/data_sources/person_for_referral_report_v2.json [hunk 2]
```diff
@@ -1323,6 +1322,254 @@
...
+                        }
+                    },
+                    "value_expression": {
+                        "type": "property_name",
+                        "property_name": "episode_type"
+                    }
+                }
+            },
+            {
+                "display_name": "adherence_schedule_id",
```
Comment (calellowitz) [inline custom/enikshay/ucr/data_sources/person_for_referral_report_v2.json]: same as above. i dont think this meets the spec. It should be `schedule_daily` -> `Daily`, `schedule_mwf` -> `Intermittent (MWF)`, and `schedule_trs` -> `Intermittent (TTS)`

**Review by calellowitz — CHANGES_REQUESTED** [review]
You appear to have missed the case definition column, as well as the separate columns for TU and PHI.
Additionally, these columns should be added to `referral_report.json`. Since the report is live right now, it would be good to create a v2 of that as well. It is also possible to do the display transforms I commented on, in the report itself if you prefer but it seems a bit cleaner to store them in the data source.

**Comment by kkrampa** [comment]
@calellowitz I prefer to do this on the report side since when a new value will be added we don't need to rebuild data source. Any benefits from keeping this in the data source?

**Comment by calellowitz** [comment]
@kkrampa report is fine

**Comment by kkrampa** [comment]
@calellowitz Updated

**Comment by calellowitz** [comment]
I think you are still missing the case definition column

**Comment by kkrampa** [comment]
Added

**Comment by calellowitz** [comment]
@kkrampa looking to push this out today. can we fix the merge conflicts

**Comment by calellowitz** [comment]
@kkrampa looks good once you fix those lint issues

**Pull request merged**